week-4/implement/password-validator.test.js

```diff
@@ -14,3 +14,36 @@ To be valid, a password must:
 
 You must breakdown this problem in order to solve it. Find one test case first and get that working
 */
+
+
+
+function isPasswordValid(password, previousPasswords = []) {
+    if (password.length < 5) {
```

Comment:
This task can be broken down into smaller steps by creating individual functions. You seem to be familiar with this approach :) ex:
`isPasswordLongerThenFiveDigits(password)` and so on so forth